packages/client/.gitignore

```
@@ -22,3 +22,7 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# SSR builds
+dist/client
```

Лучше всю папку, так как все ее содержимое не должно индексироваться

```suggestion
dist
```

---

убрала тк папка dist уже есть в игноре